app/src/main/assets/prettify-print.css

```diff
@@ -1,80 +1,13 @@
-body {
-    margin: 6px 0 6px 0;
-}
-
-.pln {
-    color: #48484C;
-}
-
-.str {
-    color: #D14;
-}
-
-.kwd {
-    color: #1E347B;
-}
-
-.com {
-    color: #93A1A1;
-}
-
-.typ {
-    color: teal;
-}
-
-.lit {
-    color: #195F91;
-}
-
-.clo,.opn,.pun {
-    color: #93A1A1;
-}
+@import "prettify-light.css";
 
-.tag {
-    color: #008;
+pre.prettyprint.linenums {
+    box-shadow: none;
 }
 
-.atn {
-    color: teal;
+li.L1,li.L3,li.L5,li.L7,li.L9 {
```

Comment:
This could also be something like 
```css
li:nth-child(even) {
    background: none;
}
```
maybe, as we want every second line to look different (should be supported by WebView in theory, not tested yet.)

Reply:
Initially I just used the same rule as the other stylesheets, but this selector works the same ([right from API 21 WebView](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child#browser_compatibility)) and also looks clearer :)